#! /bin/sh -e
## 07_noadvertising.dpatch by  <ametzler@downhill.univie.ac.at>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove advertising for cdrecord-ProDVD.

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
@DPATCH@
diff -urNad /tmp/cdrtools-2.01/cdrecord/cdrecord.c cdrtools-2.01/cdrecord/cdrecord.c
--- /tmp/cdrtools-2.01/cdrecord/cdrecord.c	Wed Jan  7 16:33:18 2004
+++ cdrtools-2.01/cdrecord/cdrecord.c	Wed Jan  7 16:35:33 2004
@@ -627,9 +627,7 @@
 			errmsgno(EX_BAD,
 			"This version of cdrecord does not include DVD-R/DVD-RW support code.\n");
 			errmsgno(EX_BAD,
-			"If you need DVD-R/DVD-RW support, ask the Author for cdrecord-ProDVD.\n");
-			errmsgno(EX_BAD,
-			"Free test versions and free keys for personal use are at ftp://ftp.berlios.de/pub/cdrecord/ProDVD/\n");
+ 			"See /usr/share/doc/cdrecord/README.DVD.Debian for details on DVD support.\n");
 		}
 		/*
 		 * Only exit if this is not the ProDVD test binary.
